### DISCUSSION TITLE
Saving large networks as high quality image

### DISCUSSION THREAD

**Discussion by ElmiraOn** [visjs/vis-network#1971] [Q&A]
Hello there, 

first of all, thank you for this amazing library, I have been using it for my projects for a while and I love the results. 

For my new project, I have a large network (300-500 nodes) and I'm trying to save the network as a high-quality image.  The other methods mentioned previously (e.g. right click + save as image) to capture the whole network, results in pixelated images where you can't even read the larger titles let alone see anything else. 

Is there any way to save large networks as high-quality images?

**Comment by ProIntegritate** [comment]
Two ways:

1. Create a virtual desktop with a custom resolution, i.e. 3200 x 3000 and place your visualization on that, then grab the screen as an image.

2. Place your visualization on the web and use a website imager that set an arbitrary window size, like 3200 x 3000 and save that to disk.
